src/explore-education-statistics-common/src/modules/table-tool/components/DataTableCaption.tsx

```diff
@@ -58,9 +59,24 @@ const DataTableCaption = ({
     publicationName,
   });
 
+  const [expanded, setExpanded] = useState<boolean>(false);
+  const captionLimit = 300;
   return (
     <>
-      <strong id={id}>{caption}</strong>
+      <strong id={id}>
+        {caption.length > captionLimit && !expanded
+          ? `${caption.substring(0, captionLimit)}...`
```

Comment:
Rather than just truncating the text, it seems to me like it would be more useful to truncate just the list of locations so that it says something like '...from Barnet, Barnsley, ... and X other locations...' when it is over a threshold size.

A caption that contains the names of every single location is not likely to be useful when there are too many. Truncating the caption as a whole will also hide the time period part of it.